src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="adduser">
+    <div class="title"> <b>Shopping Cart</b> </div> 
+    <button class="backbutton" (click)="OnCancel()">Back</button>
+    <button class="logout" (click)="logout()">Logout</button>
+    <hr>
+    <label>Enter item: </label>
+    <input #name="ngModel" type="text" [(ngModel)]="inputItem" />   
+    <div *ngIf = "name.touched && name.invalid" class="alert">Item is required</div>
+    <button style="margin-top: 10px;" (click)="addItem()"> Submit </button>
```

Comment:
Should avoid using of inline css as discussed before

Reply:
Removed the inline CSS and pushed the changes.